alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneMemory.md

```diff
@@ -0,0 +1,27 @@
+# ExtremelyHighIndividualControlPlaneMemory
+
+## Meaning
+
+This alert fires when the memory usage per an instance within control plane
+nodes exceeds 90% of total memory available per a node.  
```

Comment:
s/usage/use
s/per an instance within control plane nodes/by an instance on a control plane node
s/per a node/on that node

Reply:
+1